### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ajnadox**
Added update to new version info.

### CODE REVIEW

**Comment by ajnadox** [comment]
@ivokub would you be kind to merge this update also :)?

**Comment by ajnadox** [comment]
@ivokub this is still not merged, will you fix :) ? thanks!

**Comment by ivokub** [comment]
Thanks @ajnadox, I'll have a look and merge if possible.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,15 @@ Installation
...
 
+Update to new version
+------------
+To update the QNAP when a new tailscale releases exists:
+1. Run step 3 and then do cd tailscale-qpkg/ to enter the cloned git directory
+2. Run `git pull https://github.com/ivokub/tailscale-qpkg.git`
+3. Do step 6-9 in "How-To Build in Windows" again
+4. Then login to QNAP gui and Manually install Tailscale package in QNAP App Center again, this will overwrite the old version and maintaining settings.
+5. Done!
+
```
Comment (ivokub) [inline README.md:52]:
```suggestion
Updating
--------

When Tailscale package is already installed, then it is sufficient to install only a new version of Tailscale package in QNAP App Center. The service will restart automatically and use current configuration.

To build new package manually, rerun the commands for corresponding target (UNIX or Windows) in the updated repository.

```

**Review by ivokub — CHANGES_REQUESTED** [review]
I think the pull request makes sense in general. However, I generalized the new section for both UNIX/Windows build targets and specified that the service restarts automatically.

On another note -- please rebase on top of current master branch. There are few updates in the version numbers.

**Comment by ajnadox** [comment]
rebase on top of current master branch

**Pull request closed without merging**